guacamole/src/main/webapp/translations/en.json

```diff
@@ -394,6 +394,7 @@
     "PROTOCOL_VNC" : {
 
         "FIELD_HEADER_AUDIO_SERVERNAME" : "Audio server name:",
+        "FIELD_HEADER_CLIPBOARD_ENCODING" : "Encoding:",
```

Comment:
They should, but I don't know it.

I could only manage "clipboard" because the menu and settings had identical section headers.